Lib/ast.py

```diff
@@ -667,6 +667,20 @@ def set_precedence(self, precedence, *nodes):
         for node in nodes:
             self._precedences[node] = precedence
 
+    def get_raw_docstring(self, node):
+        """If given *node*'s body contains a docstring node, it returns
+        the docstring node. If not, it returns None"""
```

Comment:
```suggestion
        """If a docstring node is found in the body of the *node* parameter, return
        said docstring node, None otherwise."""
```